.github/ISSUE_TEMPLATE/crawler.yml

```
@@ -0,0 +1,24 @@
+name: 文章翻译
+description: 外文优秀技术、开源文章翻译
+title: （译文中文标题）
+labels:
+  - Article
+  - Translation
+body:
+  - type: input
+    id: page_link
+    attributes:
+      label: 文章网址
+      description: 一个 issue 只能填一个
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: 翻译须知
+      description: 如有其它需要注意的事项，请注明以帮助译者。
+      value: '@kaiyuanshe/team-1'
```

```suggestion
      value: '@kaiyuanshe/team'
```
根据组织的团队主页，我们可以知道与文章翻译相关的小组是：https://github.com/orgs/kaiyuanshe/teams/team